src/components/Rates/Rates.tsx

```diff
@@ -0,0 +1,186 @@
+import { useTelegram } from "../UseTelegram";
+import { useEffect, useState } from "react";
+import { BASE_URL } from "../../Constants";
+import { useParams, Link } from "react-router-dom";
+import Rating from "@mui/material/Rating";
+import StarBorder from "@mui/icons-material/StarBorder";
+import FavoriteIcon from "@mui/icons-material/Favorite";
+import FavoriteBorderIcon from "@mui/icons-material/FavoriteBorder";
+import RecordVoiceOverIcon from "@mui/icons-material/RecordVoiceOver";
+import RecordVoiceOverOutlinedIcon from "@mui/icons-material/RecordVoiceOverOutlined";
+import { useNavigate } from "react-router-dom";
+import { defaultAverage, defaultMovie, defaultYourRate } from "../constants";
+import { IMovie, IAverage, IYourRate } from "../interfaces";
+import DeleteRateButton from "../DeleteRateButton/DeleteRateButton";
+
+function Rates() {
+  const { movieId } = useParams();
+  const navigate = useNavigate();
+  const { webApp, executeMethod } = useTelegram();
+  //const { username } = webApp.initDataUnsafe?.user
+  const username = "test";
```

Comment:
don't forget to comment values for local testing and uncomment production values